### ISSUE TITLE
Get error when try in expo

### ISSUE THREAD

**Issue by Yudha17** [faizalshap/react-native-otp-verify#95]
Error: The package 'react-native-otp-verify' doesn't seem to be linked. Make sure:

- You rebuilt the app after installing the package
- You are not using Expo managed workflow
, js engine: hermes
 ERROR  Invariant Violation: "main" has not been registered. This can happen if:
* Metro (the local dev server) is run from the wrong folder. Check if Metro is running, stop it and restart it in the current project.
* A module failed to load due to an error and `AppRegistry.registerComponent` wasn't called., js engine: hermes

**Comment by BudgetalizerViraj** [comment]
same error is happening to me.

**Comment by adityab-shivamoon** [comment]
Yes, Same error, tried rebuilding the app, but not working. Please post any solution if found.

**Comment by faizalshap** [comment]
This library has not been tested in Expo

**Comment by Lekhrajk** [comment]
Facing  same issue !. Any quick resolution or what can be done ?

**Comment by pratyush0103** [comment]
facing the same issue, expo is much more in use right now, can we get expo support for this library?

**Comment by Muralidhar22** [comment]
https://docs.expo.dev/guides/local-app-development/

By running expo development build on simulator it's working fine for me, anyone facing issue once try the above steps on how to generate local app development build and run the build on simulator

**Comment by ryskin** [comment]
This only for Expo EAS, will not work with Expo Go, if you use prebuild, so just delete ios folder

**Comment by Sahil6458** [comment]
Getting Error while using CLI also.

Fix : use version 1.1.8
